gulpfile.js

```diff
@@ -296,6 +312,14 @@ function checkDir(path) {
   }
 }
 
+function getTempFile(prefix, suffix) {
+  mkdirp.sync(BUILD_DIR + 'tmp/');
```

Comment:
That is the tricky one: gulp tasks executed in parallel, so you may have multiple temp files opened. If we will try to count files of 'tmp/', there might be OS generated hidden files or just leftover after 'Ctrl+C' that might prevent removal of tmp.